rdrf/rdrf/templatetags/session_info.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag()
+def session_info():
```

Comment:
Could be in a context processor

https://github.com/eresearchqut/trrf/blob/2c6babdb8515c83d8de329dfbffed904a322ff6d/rdrf/rdrf/context_processors/context_processors.py#L1-L19

Reply:
Fixed.